content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,145 @@
+Clicking :icon:`fa-thumb-tack` :guilabel:`Pinned Messages` shows pinned messages, and clicking
+:icon:`fa-paperclip` :guilabel:`Show Attachments` shows attachments shared in the conversation.
 
-.. image:: discuss/chat-windows.png
-   :alt: View of a couple of chat window messages for Odoo Discuss.
+Click :icon:`fa-user-plus` :guilabel:`Add Users` to invite more people to the direct message
+conversation. Click :icon:`fa-cog` :guilabel:`Show Call Settings` to configure voice chat settings.
 
-.. tip::
-   When a user is mentioned, the search list (list of names) suggests values first based on the
-   task's followers, and secondly on employees. If the record being searched does not match with
-   either a follower or employee, the scope of the search becomes all partners.
+At the top of the direct message window, click the name of the direct message to change the group
+name, and choose to add a description in the adjacent :guilabel:`Add a description` field.
 
 User status
 -----------
 
-It is helpful to see what colleagues are up to and how quickly they can respond to messages by
-checking their *status*. The status is shown on the left side of a contact's name on the
-:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *chatter*.
+It is helpful to see what colleagues are up to, and how quickly they can respond to messages, by
+checking their *status*. The status is shown on the right side of a contact's name on the sidebar,
+on the *messaging menu*, and when listed in the *Chatter*.
 
 - Green = online
 - Orange = away
 - White = offline
 - Airplane = out of the office
 
-.. image:: discuss/status.png
-   :height: 300
-   :alt: View of the contacts' status for Odoo Discuss.
+Delete a direct message conversation
+------------------------------------
+
+To delete direct message conversations, click the :guilabel:`X` next to the name in the
+:guilabel:`Direct Messages` section of the sidebar.
+
+.. note::
+  This does **not** delete the :guilabel:`Direct Messages` in the conversation. The direct message
+  conversation's history is visible when another direct message with the same person, or group, is
+  created.
+
+.. _discuss/meetings:
+
+Meetings
+========
+
+In **Discuss**, :guilabel:`Meetings` are video calls. To start a meeting from the **Discuss**
+application home page, click :guilabel:`Start a meeting` in the top-left corner, and select who to
+invite to the meeting, via the :guilabel:`Invite People` drop-down window that appears. To start a
+meeting from a direct message, click the :icon:`fa-phone` :guilabel:`Start a Call` icon in the
+top-right.
+
+.. image:: discuss/meeting.png
+   :alt: View of a Meeting in Odoo Discuss.
+
+Once a meeting has been started, the buttons on the bottom of the call screen can be used to
+:icon:`fa-microphone` :guilabel:`Mute/Unmute`, :icon:`fa-headphones` :guilabel:`Deafen/Undeafen`,
+and :icon:`fa-video-camera` :guilabel:`Turn camera on/off`. Clicking the :icon:`fa-ellipsis-v`
```

Comment:
it might be useful to separate the look of the buttons for their specific use. For example, when the microphone icon is clicked and the 'Unmute' option is available, the button looks different than just a typical microphone (it's red with a line through it).

Perhaps a list with these icons and their meanings/uses could be helpful in this instance.